src/app/public/modules/title/title.service.ts

```diff
@@ -13,7 +13,9 @@ import {
 /**
  * Provides a method for setting a formatted title on the current window.
  */
-@Injectable()
+@Injectable({
+  providedIn: 'root'
+})
 export class SkyAppTitleService {
```

Comment:
For my general understanding - Why isn't `SkyAppTitleService` being provided in any modules before your change?

Reply:
This service is provided automatically in `@skyux-sdk/builder`, but we'll need to keep it in Core for backward compatibility.